code/modules/economy/vending_machines.dm

```diff
@@ -898,7 +900,8 @@
 					/obj/item/weapon/reagent_containers/food/snacks/sun_snax = 8,
 					/obj/item/weapon/reagent_containers/food/snacks/canned/appleberry = 6,
 					/obj/item/weapon/storage/box/gum = 8,
-					/obj/item/weapon/storage/box/admints = 8
+					/obj/item/weapon/storage/box/admints = 8,
+					/obj/item/weapon/reagent_containers/food/snacks/canned/ntbeans = 6
```

Comment:
A tin of beans is definitely not something you get from any 'normal' vending machine?